extensions-contrib/statsd-emitter/src/main/java/org/apache/druid/emitter/statsd/StatsDEmitterConfig.java

```diff
@@ -42,6 +42,8 @@
   private final String dimensionMapPath;
   @JsonProperty
   private final String blankHolder;
+  @JsonProperty
+  private final Boolean dogstatsd;
```

Comment:
This can be a primitive boolean

Reply:
I was just following the example from `includeHost` above. But sure, I can change this to a primitive.

Reply:
Actually, that breaks the null checks below. I'd rather keep it as is, if that's OK. I'm not really sure what all the implications of my changes would be.